Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7
+
+RUN apt-get update && apt-get upgrade -y && \
```

Comment:
We can get rid of this. I thought I had removed from all dockerfiles when I found out that it wasn't good practice

Reply:
(The whole `RUN` statement (l3,4)